pwiz_tools/Skyline/Test/TextUtilTest.cs

```diff
@@ -96,5 +97,155 @@ private IEnumerable<string> AddSuffix(string[] baseStrings, string fixText, int
             foreach (string baseString in baseStrings)
                 yield return baseString + fixText.Substring(fixText.Length - i, i++);
         }
+
+        [TestMethod]

```

Comment:
Needs a comment explaining what the test is testing for

Reply:
Closer - but it's not shuffling random strings, it's shuffling sorted strings then checking to see if we can reestablish that oder